### PR TITLE
Timed out users can not interact with buttons.

### PR DESCRIPTION
**Pull request by koala9712**
h- so why check for it

### CODE REVIEW

**Comment by koala9712** [comment]
It took only 3 forks and re-starts yes.

**Review by ooliver1 — APPROVED** [review]
h

File: cogs/help.py [hunk 1]
```diff
@@ -245,12 +238,6 @@ async def interaction_check(self, interaction: Interaction) -> bool:
         if interaction.channel.archived or interaction.channel.locked:  # type: ignore
             return False
 
-        if (
-            isinstance(interaction.user, Member)
-            and interaction.user.timeout is not None
-        ):
-            await interaction.send(timeout_message, ephemeral=True)
```
Comment (DenverCoder1) [inline cogs/help.py:252]: Is timeout_message used anywhere else? Maybe the variable can be removed?

Comment (koala9712) [reply cogs/help.py:252]: yeah, it's no logner used, removed the variable!

**Review by MaskDuck — APPROVED** [review]
h

**Comment by MaskDuck** [comment]
hhhh

**Pull request merged**